pkg/util/kubernetes/apiserver/leaderelection/leaderelection.go

```diff
@@ -156,15 +165,11 @@ func (le *LeaderEngine) EnsureLeaderElectionRuns() error {
 		return nil
 	}
 
-	le.once.Do(
-		func() {
-			go le.runLeaderElection()
-		},
-	)
+	le.StartLeaderElectionRun()
 
-	timeoutDuration := clientTimeout * 2
+	timeoutDuration := getLeaderTimeout
```

Comment:
we can probably get rid of this variable and use directly `time.After(getLeaderTimeout)`